### PR TITLE
Add POST transaction

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

**Comment by dara123** [comment]
devs please review

File: src/Entities/Transaction.php [hunk 1]
```diff
@@ -342,4 +458,6 @@ public function getCustomer()
 
         return null;
     }
+
+
```
Comment (adamaltman) [inline src/Entities/Transaction.php]: remove extra lines here

File: src/Services/TransactionService.php [hunk 2]
```diff
@@ -74,4 +74,20 @@ public function refund($transactionId, $amount)
...
+
+    /**
+     * @param array|JsonSerializable|Transaction $data
+     * @param float $amount
+     *
+     * @throws UnprocessableEntityException The input data does not valid
+     *
+     * @return Transaction
+     */
+    public function create($data)
```
Comment (adamaltman) [inline src/Services/TransactionService.php]: where is amount? or is the docblock wrong?

Comment (dara123) [reply src/Services/TransactionService.php]: sorry. docblock is wrong. will fix it

**Comment by dara123** [comment]
devs please review

**Comment by dara123** [comment]
test this please

File: src/Entities/Transaction.php [hunk 3]
```diff
@@ -249,6 +286,16 @@ public function getMethod()
         return $this->getAttribute('method');
     }
 
+    /**
+     * @param $value
+     *
+     * @return Transaction
+     */
+    public function setMethod($value)
```
Comment (adamaltman) [inline src/Entities/Transaction.php]: This seems wrong? 

Where is the paymentInstrument set?

**Pull request merged**